wideq.py

```diff
@@ -679,6 +679,40 @@ def enum_name(self, key, value):
         options = self.value(key).options
         return options[value]
 
+    @property
+    def monitor_type(self):
+        """Get type of monitoring return data.
+        """
+
+        if self.data['Monitoring']['type'] == 'BINARY(BYTE)':
+            return 1
+        else:
+            return 0
+
+    def decode_monitor_binary(self, data):
+        """Decode binary encoded status data.
+        """
+
+        decoded = {}
+        for item in self.data['Monitoring']['protocol']:
+            value = 0
+            for i in range(item['startByte'], item['startByte'] + item['length']):
+                value = value * 256 + data[i]
```

Comment:
FWIW, it might be clearer what's going on here (i.e., you're decoding a value one byte at a time) if you were to use `value << 8` instead of `value * 256`.